File: tests/Generator/resource/swift/complex/DefinitionType.swift
// Base definition type
class DefinitionType: Codable {
    var deprecated: Bool?
    var description: String?
    var _type: String?

    enum CodingKeys: String, CodingKey {
        case deprecated = "deprecated"
        case description = "description"
        case _type = "type"
    }
}

File: tests/Generator/resource/python/complex/definition_type.py
from pydantic import BaseModel, Field, GetCoreSchemaHandler, Tag
from pydantic_core import CoreSchema, core_schema
from typing import Any, Dict, Generic, List, Optional, TypeVar, Annotated, Union


# Base definition type
class DefinitionType(BaseModel):
    deprecated: Optional[bool] = Field(default=None, alias="deprecated")
    description: Optional[str] = Field(default=None, alias="description")
    type: Optional[str] = Field(default=None, alias="type")
    pass


